secrets/hcvault/secret.go

```diff
@@ -21,8 +22,8 @@ func (c *HCVaultSecretStore) GetSecretAsByte(root string, path string) ([]byte,
 	return secretDataByte, nil
 }
 
-func (c *HCVaultSecretStore) GetSecret(root string, path string) (map[string]interface{}, error) {
-	return c.GetSecretWithPath(fmt.Sprintf("%s/data/%s", root, path))
+func (c *HCVaultSecretStore) GetSecret(path ...string) (map[string]interface{}, error) {
+	return c.GetSecretWithPath(strings.Join(path, "/data/"))
```

Comment:
I made that change like YOLO. That needs to be verified with you @daniel1302.

Reply:
I have fixed it already in another PR